Dockerfile

```diff
@@ -42,35 +50,48 @@ RUN yarn && yarn cache clean --force
 USER root
 RUN apt-cleanup.sh build-essential
 
-COPY --chown=appuser:appuser docker-entrypoint.sh /entrypoint/docker-entrypoint.sh
-ENTRYPOINT ["/entrypoint/docker-entrypoint.sh"]
+COPY --chown=appuser:appuser . .
```

Comment:
Maybe using full paths would be safer and easier to read. I think this would be then:
`COPY --chown=appuser:appuser . /app/.`

Reply:
yes, let's do this 👍